tvision/makes/config.nmk

```diff
@@ -9,15 +9,22 @@
 
 !ifdef DEBUG
 # Options to debug
-CFLAGS   = /nologo -D_WIN32 -DFOR_EDITOR /EHsc -D_USE_32BIT_TIME_T -D_CRT_SECURE_NO_DEPRECATE /Zi
 LDFLAGS  = /DEBUG
+!ifdef WIN64
```

Comment:
The purpose of the changes to this file is to make it compatible with both 64-bit and 32-bit compiles, so the developer does not need to edit this file for each case.  The default is to do a 32-bit build.  If the developer wants to do a 64-bit build, they just need to add WIN64=1 onto the end of the invocation of nmake.  If you want to make the 64-bit build the default, we can easily change this around for that.